src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1104,7 +1104,8 @@ export function logScalePossible(
         // molecular profile
         return !!(
             axisSelection.dataSourceId &&
-            logScalePossibleForProfile(axisSelection.dataSourceId)
```

Comment:
Ok, this relates to another PR that I have filed for changes to the PlotsTab that includes a rework of log-scale evaluation (treatment profiles can be log-transformed). Since I have split the project in multiple PRs this is one of the consequences of this. Would it be better to revert these lines into the original state?

Reply:
Yes please, since logScalePossibleForProfile literally does the exact computation.

Reply:
I looked into the logScalePossibleForProfile() function. The reason for isolating the logTransformation logic into a separate function is that the CoExpressionViz.tsx component duplicated this logic in the showLogScaleControls() function. I think therefore having an isolated  logScalePossibleForProfile function is justified, also considering that treatment profiles will be added to this logic. Possible the logScalePossibleForProfile() function should move to a more suited class? Could you propose one?

Reply:
Sorry, I'm confused -- I don't understand why we'd make this change here since it's literally the same logic. Are you planning to change the functionality of logScalePossibleForProfile?

Reply:
What I mean is that the logic of logScalePossible():

```
return !(/zscore/i.test(profileId)) &&
        /rna_seq/i.test(profileId);
```
is duplicated in the CoExpressionViz.tsx component. What I did was isolate the logic into the logScalePossibleForProfile() helper function and used this helper function in CoExpressionViz.tsx:

```    
@computed get showLogScaleControls() {
        const profileXId = this.props.profileX.molecularProfileId.toLowerCase();
        const profileYId = this.props.profileY.molecularProfileId.toLowerCase();
        return (
            (logScalePossibleForProfile(profileXId) || logScalePossibleForProfile(profileYId))
        );
    }
```
The is refactoring is even more important considering that will update the logScalePossible() logic for treatments in the plots tab. So the center of the debate is that your statement:

>...and this function and logScalePossibleForProfile are both in the same file. 

is incorrect and that I should answer your question:
>Are you planning to change the functionality of logScalePossibleForProfile?

with "yes".

Reply:
I see `logScalePossibleForProfile` in this file on line 1095. You may not have realized that was there, which is why there is confusion. In light of that, do you still want to write out the logic this way?

Reply:
I have reverted the log-scale evaluation to the rc state.

Reply:
I still see it expanded here on lines 1107 and 1108

Reply:
I am losing track of your intentions with this comment. These lines are what is currently in rc branch. How can your comment on the expanded line relate to my PR?

Reply:
Look at lines 1107 and 1108 - they are green, indicating they have been added by this PR. Im asking that you undo all changes in this file `PlotsTabUtils.tsx`

Reply:
Sorry for this ... I do not know why this was the case. Lines reverted back  to rc state.